### PR TITLE
Adding config yaml for externalising tokens

### PR DESCRIPTION
**Pull request by Balasundaram**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -33,8 +36,8 @@
         conn = httplib.HTTPSConnection("api.pushover.net:443")
         conn.request("POST", "/1/messages.json",
         urllib.parse.urlencode({
-        "token": "<REPLACE_API_TOKEN>",
-        "user": "<REPLACE_USER_KEY>",
+        "token": config["API_TOKEN"],
+        "user": config["USER_TOKEN"],
```
Comment (satrox28) [inline main.py]: It should be USER_KEY

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 requests
 http.client
+yaml
```
Comment (satrox28) [inline requirements.txt]: pyyaml instead of yaml

**Review by satrox28 — COMMENTED** [review]
It should be USER_KEY

File: requirements.txt (hunk 2 above)
Comment (Balasundaram) [reply requirements.txt]: Sure thing

Comment (Balasundaram) [reply requirements.txt]: Done Sathish, added the changes

**Review by satrox28 — APPROVED** [review]

**Pull request merged**